version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.8.2-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.8.3-SNAPSHOT"
```

Comment:
Should this be a bigger upgrade, because of number of depency changes. perhaps 0.9.0 ?